python/samples/documentation_examples/prompts.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+from service_configurator import add_ai_service
+
+import semantic_kernel as sk
+
+
+async def main():
+    # Initialize the kernel
+    kernel = sk.Kernel()
+
+    # Add the service to the kernel
+    # use_chat: True to use chat completion, False to use text completion
+    kernel = add_ai_service(kernel=kernel, use_chat=True)
+
+    request = input("Your request: ")
+
+    # 0.0 Initial prompt
+    prompt = f"What is the intent of this request? {request}"
+    print("0.0 Initial prompt")
+    print("-------------------------")
+    prompt_function = kernel.create_function_from_prompt(
+        function_name="sample_zero", plugin_name="sample_plugin", prompt=prompt
```

Comment:
does this name refer to a sample number?

Reply:
Just an arbitrary name, corresponding to the example... 0.0, 1.0, 2.0, etc.